README.md

```diff
@@ -3,16 +3,16 @@
 [![Build Status](https://travis-ci.org/ksonnet/ksonnet.svg?branch=master)](https://travis-ci.org/ksonnet/ksonnet)
 [![Go Report Card](https://goreportcard.com/badge/github.com/ksonnet/ksonnet)](https://goreportcard.com/report/github.com/ksonnet/ksonnet)
 
-A tool for managing Kubernetes resources as code.
+ksonnet provides a framework for you to write, share, and deploy Kubernetes application manifests. With its CLI, you can generate a complete application from scratch in only a few commands, or manage a complex system at scale.
```

Comment:
- [ ] I stumble over the "for you to" part on each reading.  
  Consider:  
  > _ksonnet_ is a framework for writing, sharing, and deploying ...